Microsoft.Toolkit.Uwp.UI.Controls.Primitives/Microsoft.Toolkit.Uwp.UI.Controls.Primitives.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+	<PropertyGroup>
+		<TargetFrameworks>uap10.0.17763</TargetFrameworks>
+		<Title>Windows Community Toolkit Controls</Title>
+		<Description>
+			This library provides XAML templated controls. It is part of the Windows Community Toolkit.
```

Comment:
```suggestion
			This library provides controls for use in XAML which don't provide their own style. It is part of the Windows Community Toolkit.
```

I don't think we can call these templated controls anymore as they're not providing templates... :)